ts/features/payments/checkout/saga/networking/handleWalletPaymentCreateTransaction.ts

```diff
@@ -44,7 +22,9 @@ export function* handleWalletPaymentCreateTransaction(
     );
 
     if (E.isLeft(newTransactionResult)) {
-      handleError(paymentAnalyticsData, action.payload.onError);
+      if (action.payload.onError) {
```

Comment:
Is this necessary?
I think we can omit the `return` and allow the `finally` block to execute regardless of whether an error occurred.
What are your thoughts?